inputgraph/loader.go

```diff
@@ -364,13 +364,54 @@ func (l *loader) handleIf(ctx context.Context, ifStmt spec.IfStatement) error {
 
 func (l *loader) handleFor(ctx context.Context, forStmt spec.ForStatement) error {
 	l.hashForStatement(forStmt)
-	err := l.loadBlock(ctx, forStmt.Body)
+
+	opts := commandflag.ForOpts{
+		Separators: "\n\t ",
```

Comment:
I don't like how this value is now pre-initialized in two locations,

Maybe create a function in commandflag along the lines of

```
func InitForOptDefauls(*fo *ForOpts) {
  fo.Separators = "\n\t"
}
```